### ISSUE TITLE
Is it a duplicate of https://github.com/racodond/sonar-css-plugin?

### ISSUE THREAD

**Issue by v-karbovnichy** [kalidasya/sonar-css-plugin#9]

**Comment by v-karbovnichy** [comment]
I've also raised an issue at that repo https://github.com/racodond/sonar-css-plugin/issues/83
All comments from it are also meant to be addressed here

**Comment by v-karbovnichy** [comment]
How can it be that "official" repo has no answer for 21 days, but "unofficial" gives proper explanations for this?

Hello, anyone?

**Comment by kalidasya** [comment]
well its two repo, you have your solution in one why do you expect it here as well?
And dont forget, you free to create MR-s with your code changes!

**Comment by kalidasya** [comment]
I created this plugin long ago so people can analyse css projects with sonarqube, in the last few years I do not work with CSS anymore, David does and/or his clients do, so you can expect more activity there. If you do not plan to give back to the community I would suggest to use David's repo, as I have no time to keep this plugin up to date, especially when David is doing a good job on updating the plugin. 
If you wonder why is this the official and not that, is that this is guaranteed (while I am the owner) to be available in the future in this state while the other is not (as it went offline before). This not necessary mean it will happen, just in case.

**Comment by v-karbovnichy** [comment]
Ok, I do not know the deep details of relationship between @racodond  and SonarSource SA. And I do not want to judge it.

@racodond, I kindly suggest you also read this. Let me express my vision from a nonbiased user perspective.

A month ago our team tried to use SonarQube for our project for for 5 languages total. CSS plugin (taken from SQ marketplace, v.3) gave us strange results for SCSS files, many false positives and some plain wrong suggestions for SCSS (I believe it was checking CSS ruleset against SCSS code). Although the description for the plugin on the marketplace contained support for SCSS. We have fixed number of issues, but it was very big number to address (>500). I found these 2 repos trying to search for root cause, but didn't pay attention to release numbers, and also in that time a deadline came. So we returned to this problem just today, installed a jar of 4.1 (unofficial one :) and found that it is producing far more relevant results and also correctly handling SCSS syntax.

So as you can see, we had invested in proper configuration of SonarQube and for 3 first weeks had opinion that it will not fit our needs because it gives just plain wrong advices. This situation for us affected badly the SonarQube reputation as a product. So I believe it is bad for you, @kalidasya.

Also this situation will affect @racodond repo. I believe we are not the single team that had problems with version that is in marketplace (btw latest!). So some of these users will go and open issues like I did. But most of them will not notice that they will open issues against wrong repo. And they will blame you, David, for the binaries they have installed from marketplace.

So let me express my view of possible solutions for the situation
* just close these issue, forget about it and go ahead. Ok, I understand, life is life, I don't know all the reasons and decisions. We all have other work to do. No complaints.
* make plugin names distinguishable from each other. Right now they have single name, and it is wrong. This is actually the root cause of confusion. If some products forked in public and go live in parallel, they should have different names from the fork point. Make both plugins be in marketplace and let the community decide what to install. So @kalidasya you will be defended from @racodond actions, and you @racodond will have direct marketplace entry and wider community.
* unite again. At least think about it. @kalidasya, you can simply maintain a mirror of @racodond repo if you think it can go down again. Git holds all possible scenarios nowadays.

If you had read so far, I greatly appreciate it.

**Comment by kalidasya** [comment]
@v-karbovnichy this all understandable and makes sense, the problem is, that @racodond does not want to be bound to SonarSource in any way he wants freedom in what and how he releases, so his plugin will not be available in the marketplace. The only thing I can do is to remove the css plugin from the marketplace and deprecate this repo.
